remoteappmanager/docker/container_manager.py

```diff
@@ -62,7 +54,11 @@ def start_container(self, user_name, image_name, volumes=None):
             The name of the user
         image_name: string
             A string identifying the image name.
-        volumes: dict
+        mapping_id: str
+            A generic id used to recognize the container. It is not unique
```

Comment:
I would just state that this id is expected to be unique (and persistent) for a specific combination of docker image (i.e. application) and setup (i.e. configuration).